content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md

```diff
@@ -19,11 +19,11 @@ With the Palette Nested Cluster option, you can deploy additional Kubernetes clu
 
 There are certain advantages to use a nested cluster environment. For example, you are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.
 
-Try the steps below to [enable](/clusters/nested-clusters/cluster-quickstart#enablinganestedclusteronahostcluster) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a managed Nested Cluster from within the Palette orchestrator.
+Try the steps below to [enable](/clusters/nested-clusters/cluster-quickstart#enablinganestedclusteronahostcluster) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a managed Nested Cluster from within Palette.
 
 ## Prerequisites
 
-- You will need to have access to a Palette Account. Refer to the [Getting Started with Palette](/getting-started) page.<p></p><br />
+- You will need to have access to a Palette Account. Refer to the [Getting Started with Palette](/getting-started) page, and sign up for free [here](https://www.spectrocloud.com/get-started/).<p></p><br />
 
 - Ensure you have a functioning [cluster](/clusters/new-clusters) already configured. During the cluster creation, you will be able to configure it as a Host Cluster.
```

Comment:
"During the cluster creation, you will be able to configure it as a Host Cluster."

here lets specify "During the nested cluster creation, you will be able to configure it as a Host Cluster.",  

-else it can confuse with host cluster creation @Siboney1

Reply:
Tyler updated this section to clarify.